build_docs.pl

```diff
@@ -69,6 +69,12 @@ BEGIN
 forever in honor of our success but otherwise doesn't do anything.
 MSG
 }
+if ( $Opts->{direct_html} ) {
+   say <<MSG
+The direct_html migration is complete! --direct_html will emit this message
+forever in honor of our success but otherwise doesn't do anything.
+MSG
+}
```

Comment:
😍

Reply:
I don't want to drop the parameter or else it'll break everyone that added it1